codex-rs/core/tests/test_support.rs

```diff
@@ -76,3 +76,24 @@ pub fn load_sse_fixture_with_id(path: impl AsRef<std::path::Path>, id: &str) ->
         })
         .collect()
 }
+
+#[allow(dead_code)]
```

Comment:
May we need an "integration tests util" crate so we don't have to do this `dead_code` thing, which is distracting/misleading.